lib/cookies.js

```diff
@@ -39,7 +39,7 @@ function deleteCookie(name) {
 }
 
 module.exports = {
-  setCookie: setCookie,
-  getCookie: getCookie,
-  deleteCookie: deleteCookie
+  set: setCookie,
+  get: getCookie,
+  delete: deleteCookie
```

Comment:
Are these cookie functions part of our public API contract?

Reply:
No, we don't expose them.